R/catmaid_annotations.R

```diff
@@ -280,10 +274,7 @@ possibly.numeric <- function(x) {
 #' Lock or unlock a CATMAID neuron reconstruction
 #'
 #' @description  Lock or unlock a CATMAID neuron reconstruction by adding or removing a 'locked' annotation to a set of skeleton IDs (skids). A locked neuron cannot be edited until it is unlocked.
-#' @param skids the skeleton IDs neurons you wish to lock / unlock
-#' @param pid project id. Defaults to 1
-#' @param conn CATMAID connection object, see ?catmaid_login for details
-#' @param ... methods passed to catmaid_fetch
+#' @inheritParams read.neuron.catmaid
```

Comment:
@alexanderbates FYI you can save a bit of typing by inheriting the docs for these 4 arguments from `read.neuron(s).catmaid` (they are jointly documented so both `skid` and `skids` are defined there).